kubejs/server_scripts/gregtech/recipes.js

```diff
@@ -66,6 +66,15 @@ const registerGTCEURecipes = (event) => {
 
 	//#endregion
 
+    event.recipes.gtceu.centrifuge('beets_to_sugar')
```

Comment:
nitpick, ideally this should follow the recipeID pattern

'tfg:centrifuge/beets_to_sugar'

Reply:
if you write 'tfg:beets_to_sugar' on a gt machine, it'll automatically add the 'centrifuge/' bit for you!